scss/3-global/_site.scss

```diff
@@ -64,16 +69,25 @@ header.navbar.navbar-inverse {
 .navbar-header {
 		position: relative;
 		top: -6px;
+    font-weight: bolder;
+    font-family: 'Josefin Sans', sans-serif;
+    .nudge-left {
+      position: relative;
+      right: 6px;
+      font-size: 24px;
+    }
 }
 
 ::selection {
     text-shadow: none !important;
     background: $selection-color;
+    color: #222;
```

Comment:
How about adding `colors.scss` and defining a set of named color variables there?